pkgs/tools/misc/popsicle/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
+, fetchFromGitHub
+, rustc
+, cargo
+, rustPlatform
+, pkgconfig
+, dbus
+, glib
+, cairo
+, pango
+, atk
+, gdk-pixbuf
+, gtk3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "popsicle";
+  version = "b02ebf5f2e6c18777453ca9a144d69689a6fa901";
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = pname;
+    rev = version;
+    sha256 = "03ilhvnr4mwy7b8bipp616h16m2ilxzxz2zjpkzy3afwvh9bz1mx";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    sha256 = "0kvv30pbcjsgpyyam2hlpjxh6vz222dmfjcdv0nrhh0441j1x4xp";
+  };
+
+  nativeBuildInputs = [
+    pkgconfig
+    dbus
+    glib
+    cairo
+    pango
+    atk
+    gdk-pixbuf
+    gtk3
+  ] ++ (with rustPlatform; [
+    cargoSetupHook
+    rust.cargo
+    rust.rustc
+  ]);
+
+  installPhase = ''
+    make install PREFIX=$out DESTDIR=$out
+
+    mv $out/usr/local/* $out
+    rm $out/usr  -rf
+  '';
+
```

Comment:
Missing meta section, for e.g. the description, homepage, license, maintainers, etc...

Reply:
Added thank you!